publication-commons/src/main/java/no/unit/nva/publication/rightsretention/RightsRetentionsUtils.java

```diff
@@ -0,0 +1,75 @@
+package no.unit.nva.publication.rightsretention;
+
+import static no.unit.nva.model.associatedartifacts.RightsRetentionStrategyConfiguration.NULL_RIGHTS_RETENTION_STRATEGY;
+import static no.unit.nva.model.associatedartifacts.RightsRetentionStrategyConfiguration.OVERRIDABLE_RIGHTS_RETENTION_STRATEGY;
+import static no.unit.nva.model.associatedartifacts.RightsRetentionStrategyConfiguration.RIGHTS_RETENTION_STRATEGY;
+import static nva.commons.core.attempt.Try.attempt;
+import java.util.Set;
+import no.unit.nva.model.associatedartifacts.CustomerRightsRetentionStrategy;
+import no.unit.nva.model.associatedartifacts.FunderRightsRetentionStrategy;
+import no.unit.nva.model.associatedartifacts.NullRightsRetentionStrategy;
+import no.unit.nva.model.associatedartifacts.OverriddenRightsRetentionStrategy;
+import no.unit.nva.model.associatedartifacts.RightsRetentionStrategy;
+import no.unit.nva.model.associatedartifacts.RightsRetentionStrategyConfiguration;
+import no.unit.nva.model.associatedartifacts.file.File;
+import no.unit.nva.publication.commons.customer.Customer;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.BadRequestException;
+
+public class RightsRetentionsUtils {
+    public static RightsRetentionStrategy getRightsRetentionStrategy(Customer customer, File file,
+                                                                     String username)
+        throws BadRequestException {
+        var configuredRightsRetention = getConfigFromCustomerDto(customer);
+        var overridedRightsRetention = file.getRightsRetentionStrategy();
```

Comment:
I would avoid naming the RRS from the client `overridedRightsRetention`. `actualRightsRetention` or `fileRightsRetention` is clearer.